test_cases/person_class_tests.py

```diff
@@ -0,0 +1,28 @@
+import unittest
+from person_class import *
```

Comment:
Avoid **import *** as because it puts a lot of stuff into your namespace (might shadow some other object from previous import and you won't know about it). Always use explicit imports